sdlib/d/gc/page.d

```diff
@@ -937,27 +952,34 @@ unittest allocPages {
 	auto e0 = checkAllocPage(1, true);
 	auto e1 = checkAllocPage(2, true);
 	assert(e1.address is e0.address + e0.size);
+	assert(filler.usedPages == 3);
 
 	auto e0Addr = e0.address;
 	filler.freePages(e0);
+	assert(filler.usedPages == 2);
 
 	// Do not reuse the free slot is there is no room.
 	auto e2 = checkAllocPage(3, true);
 	assert(e2.address is e1.address + e1.size);
+	assert(filler.usedPages == 5);
 
 	// But do reuse that free slot if there isn't.
 	auto e3 = checkAllocPage(1, false);
 	assert(e3.address is e0Addr);
+	assert(filler.usedPages == 6);
 
 	// Free everything.
 	filler.freePages(e1);
 	filler.freePages(e2);
 	filler.freePages(e3);
+	assert(filler.usedPages == 0);
 
 	// Check a wide range of sizes.
 	foreach (pages; 1 .. 2 * PagesInBlock) {
 		auto e = checkAllocPage(pages, true);
+		assert(filler.usedPages == pages);
```

Comment:
You repeat the pattern of calling checkAllocPages and then checking the number of used pages. IMO you'd be better of updating checkAllocPages to do this consistently.

Reply:
Fixed